protocol/governance/contracts/Proxy.sol

```diff
@@ -4,9 +4,8 @@ pragma solidity >=0.8.11 <0.9.0;
 import {UUPSProxyWithOwner} from "@synthetixio/core-contracts/contracts/proxy/UUPSProxyWithOwner.sol";
 
 contract Proxy is UUPSProxyWithOwner {
-    // solhint-disable-next-line no-empty-blocks
     constructor(
         address firstImplementation,
         address initialOwner
-    ) UUPSProxyWithOwner(firstImplementation, initialOwner) {}
+    ) UUPSProxyWithOwner(firstImplementation, initialOwner) {} // solhint-disable-line no-empty-blocks
```

Comment:
even changing a comment will result in a new proxy deployed => change of proxy address